pxtblocks/blocklyloader.ts

```diff
@@ -566,6 +583,17 @@ namespace pxt.blocks {
                     builtinBlocks[fn.attributes.blockId].symbol = fn;
                 } else {
                     let pnames = parameterNames(fn);
+                    // use slider for min and max number
+                    if (fn.attributes.min && fn.attributes.max) {
+                        // change to numberMinMax only if min and max are defined
+                        for (let i = 0; i < fn.parameters.length; i++) {
+                            let pr = fn.parameters[i];
+                            if (fn.attributes.min[pr.name] && fn.attributes.max[pr.name]) {
+                                fn.parameters[i].type = "numberMinMax";
+                                pnames[pr.name].type = "numberMinMax";
+                            }
```

Comment:
Here was necessary to change the block type, because in PXT a variable of type number automatically is linked with a math_number block.